process-document/src/main/java/com/ritense/processdocument/autoconfigure/ProcessDocumentAutoConfiguration.java

```diff
@@ -109,7 +110,9 @@ public CamundaProcessJsonSchemaDocumentAssociationService processDocumentAssocia
     }
 
     @Bean
+    @ProcessBean
     @ConditionalOnMissingBean(DocumentVariableDelegate.class)
+    @Deprecated(since = "11.0.0", forRemoval = true)
```

Comment:
We've not been doing this right, but can you reference the new alternative here?

Reply:
Marijn suggested 4 days ago to change this to annotation. What is the new alternative?

Reply:
The alternative is the new DocumentDelegateService that is introduced with this PR

Reply:
Sorry, I might not have been clear. My comment was on the `@Deprecated` annotation. We've been getting comments that we're deprecating stuff without offering an alternative. Unfortunately, the annotation doesn't provide a way to state the alternative, so that should be added in a Javadoc.